src/ApiPort.VisualStudio/source.extension.vsixmanifest

```diff
@@ -1,19 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="00000000-0000-0000-0000-000000000000" Version="1.3.0" Language="en-US" Publisher="Microsoft" />
+    <Identity Id="00000000-0000-0000-0000-000000000000" Version="1.4.0" Language="en-US" Publisher="Microsoft" />
     <DisplayName>SEE DESCRIPTION</DisplayName>
     <Description xml:space="preserve">NOTE: Modify src/ApiPort.Vsix/source.extension.vsixmanifest.that exists because that is the one we deploy.</Description>
   </Metadata>
   <Installation>
-    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Pro" />
-    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Ultimate" />
-    <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.Premium" />
+    <InstallationTarget Version="[14.0,16.0)" Id="Microsoft.VisualStudio.Pro" />
```

Comment:
I looked through the [vsix-extension-schema docs](https://docs.microsoft.com/en-us/visualstudio/extensibility/vsix-extension-schema-2-0-reference#installation-element) and it appears that it is just an enumeration... there doesn't appear to be a heirarchy.